### PR TITLE
WFCORE-930 help files for embedded host controller

### PR DESCRIPTION
**Pull request by luck3y**
Add help files for embed-host-controller and stop-embedded-host-controller CLI commands.

### CODE REVIEW

**Comment by luck3y** [comment]
Retest this please

**Comment by luck3y** [comment]
retest this please

File: cli/src/main/resources/help/embed-host-controller.txt [hunk 1]
```diff
@@ -0,0 +1,56 @@
+SYNOPSIS
+
+   embed-host-controller [-c=config_file || --domain-config=config=domain_config_file]
```
Comment (bstansberry) [inline cli/src/main/resources/help/embed-host-controller.txt]: Should be

... || --domain-config=domain_config_file

I think the text you have was a carry-forward of a typo I made in embed-server.txt

... || --server=config=config_file

The first '=' there should be a '-'. Can you fix that in this PR as well? Thanks.

File: cli/src/main/resources/help/embed-host-controller.txt [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+                         [--host-config=config=host_config_file]
```
Comment (bstansberry) [inline cli/src/main/resources/help/embed-host-controller.txt]: Same issue here.

File: cli/src/main/resources/help/embed-host-controller.txt [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+                     Must be specified if the environment variable
+                     JBOSS_HOME is not set. In a modular classloading
+                     environment it is assumed the CLI is running from
+                     the host controller installation itself and the JBOSS_HOME
+                     environment variable must be set.
+
+ --domain-config   - Name of the domain configuration file to use
+                     (default is "domain.xml")
+                     (Same as -c)
+
```
Comment (bstansberry) [inline cli/src/main/resources/help/embed-host-controller.txt]: --host-config description is missing.

**Comment by luck3y** [comment]
Oops, yep sorry about that.

**Pull request merged**